examples/webgl_loader_viewer.html

```diff
@@ -0,0 +1,410 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>three.js webgl - model viewer with OrbitControls</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+	<style>
+	body {
+		font-family: Monospace;
+		background-color: #000;
+		color: #fff;
+		margin: 0px;
+		overflow: hidden;
+	}
+	#info {
+		color: #fff;
+		position: absolute;
+		top: 10px;
+		width: 100%;
+		text-align: center;
+		z-index: 100;
+		display:block;
+	}
+	#info a, .button { color: #f00; font-weight: bold; text-decoration: underline; cursor: pointer }
+	</style>
+</head>
+
+<body>
+	<div id="info">
+		<a href="http://threejs.org" target="_blank">three.js</a> - Model Viewer test with OrbitControls
+		<br>
+		select model:
+		<select id="fileModel" onchange="reloadModel(this.value);">
+			<option value="models/babylon/skull.babylon">models/babylon/skull.babylon</option>
+			<option value="obj/female02/female02.obj">obj/female02/female02.obj</option>
+			<option value="obj/male02/male02.obj">obj/male02/male02.obj</option>
+			<option value="obj/walt/WaltHead.obj">obj/walt/WaltHead.obj</option>
+			<option value="models/awd/simple/simple.awd">models/awd/simple/simple.awd</option>
+			<option value="models/collada/monster/monster.dae">models/collada/monster/monster.dae</option>
+			<option value="models/collada/kawada-hironx.dae">models/collada/kawada-hironx.dae</option>
+			<option value="models/fbx/xsi_man_skinning.fbx">models/fbx/xsi_man_skinning.fbx</option>
+			<option value="models/gltf/duck/glTF/duck.gltf">models/gltf/duck/glTF/duck.gltf</option>
+			<option value="models/json/teapot-claraio.json">models/json/teapot-claraio.json</option>
+			<option value="models/skinned/marine/marine_anims_core.json">models/skinned/marine/marine_anims_core.json</option>
+			<option value="models/assimp/jeep/jeep.assimp.json">models/assimp/jeep/jeep.assimp.json</option>
+			<option value="models/pcd/Zaghetto.pcd">models/pcd/Zaghetto.pcd</option>
+			<option value="models/sea3d/mascot.tjs.sea">models/sea3d/mascot.tjs.sea</option>
+			<option value="models/vrml/house.wrl">models/vrml/house.wrl</option>
+		</select>
+		<button onclick="toggleDiv('debug')">Toggle Debug output</button>
+		<br>
+		Custom model:
+		<input type="text" name="custompath" size="50" id="customModel" value="http://people.sc.fsu.edu/~jburkardt/data/obj/shuttle.obj">
+		<button onclick="reloadModel(getElementById('customModel').value)">Load Custom Model</button>
```

Comment:
@bchevalier yes, I just saw that with html5 while it is not possible to get a file full path, it is possible to select a local file and get its data.

@mrdoob yes, thank you, I should have look at it earlier; I just tried it, lol :). 

Having a smaller viewer example to just load and view models could be possibly useful? I will study the editor code to see what I can do better.